extensions/src/main/java/it/ministerodellasalute/immuni/extensions/http/TrafficAnalysisPreventionHeadersInterceptor.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+ * Please refer to the AUTHORS file for more information.
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package it.ministerodellasalute.immuni.extensions.http
+
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import java.io.IOException
+
+/**
+ * TrafficAnalysisPreventionHeadersInterceptor is a [OkHttpClient] network interceptor that
+ * remove some default added http headers.
```

Comment:
I'd change `remove some default added http headers` to `removes some of the http headers added by default`